### DISCUSSION TITLE
Copilot stops working sometimes

### DISCUSSION THREAD

**Discussion by vineet-codes256** [community/community#30801] [Copilot Conversations]
Copilot just Stops working sometimes, It would usually start working again after reloading vs code or reinstalling or re-authenticating it.

**Comment by dmaly56** [comment]
same issue here
i sometimes have to reinstall it

**Reply by MichaelReaNZ** [reply]
Mine has stopped working completely on VS2022 on Vs code it's fine.

**Reply by dmaly56** [reply]
i had vscode and issues :/

**Reply by davecheney** [reply]
@dmaly56 
i'm sorry you are having difficulties.  Can you please tell me a bit more about what you tried, and what happened. Thank you.

**Comment by davecheney** [comment]
@vineet-codes256 i'm sorry you are having difficulties. Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by vineet-codes256** [reply]
`[INFO] [ghostText] [2022-08-26T11:08:49.631Z] Found inline suggestions locally
[INFO] [ghostText] [2022-08-26T11:08:49.631Z] Filtered out solution matching next line
`
![image](https://user-images.githubusercontent.com/87371365/186891345-aa42b5b2-56b7-4e97-af1c-fb418e9c9003.png)

It says ** Filtered out solution matching next line** 
Sometimes when inside a nested loop it will not generate the end bracket for the child loop because the parent's end bracket was already there in the next line.

I'll update if I find something else.

**Reply by davecheney** [reply]
It looks like copilot is working. Sometimes it doesn't have any suggestions to offer

**Comment by cpgames** [comment]
this is a constant annoying issue for me. After using it for a bit, it would typically decide its work day is over and stop offering any suggestions at all. The only solution is restarting Visual Studio. It happens every time 1-2 hours into using it. I have it on two different machines with different VS installs, happens on both. I think it's likely caused by reloading solution.

**Comment by serlogger** [comment]
As a beginner with Copilot, I was puzzled at first as to why it would sometimes work and sometimes not. Then I realized that I had been sometimes trying to use it in the middle of a line instead of at the _end_ of it. After this, I realized it has always been working for me. Copilot had been just waiting for lines to be completed (and not sections inside a "completed" line).
